tests/org.jboss.tools.vpe.preview.editor.test/src/org/jboss/tools/vpe/preview/editor/test/util/TestUtil.java

```diff
@@ -57,7 +56,8 @@
      * @return {@link VpvEditorController}
      */
     public static VpvEditorController getVpvEditorController(JSPMultiPageEditor part) {
-        VpvEditorPart visualEditor = (VpvEditorPart) part.getVisualEditor();
+    	VpvEditorPart visualEditor = (VpvEditorPart) part.getVisualEditor();
+    	visualEditor.createVisualEditor(); //this needs to be done to make sure visualEditor is created
```

Comment:
This broke PreviewReloadTest - visualEditor might be null in some cases - depending which VPV page was active